packages/jwt-verifier/test/spec/verifier.spec.js

```diff
@@ -147,6 +147,63 @@ describe('Jwt Verifier', () => {
       )));
   });
 
+  it('should only allow includes operator for custom claims', () => {
```

Comment:
Why no test of the happy path?

Also (untested) - are these tests going to handle the async well?  You're not using done() nor declaring the test function to be async, so I'd think the actual expects() would potentially run AFTER the test runner was complete.

Reply:
I was just mimicking the existing test for assertClaims. Maybe they all need a reworking?

Reply:
I experimented with testing the happy path, but it requires actually getting information into a real access token.

I'd rather write some unit tests with fixtures rather than relying on the presence of a real okta org. I'll get on that.